app/models/assignment_participant.rb

```diff
@@ -287,4 +290,16 @@ def stage_deadline
     end
     stage
   end
+
+  # E1834 Fall 18
+  def is_round_valid_for_mail?
+    topic_id = SignedUpTeam.topic_id(self.parent_id, self.user_id)
+    return assignment.review_deadline(topic_id, assignment.num_review_rounds)
+  end
+
+  # #E1834 Fall 18
```

Comment:
Remove commented-out code.

Reply:
We have removed the commented code as per suggestion.